openpype/hosts/max/api/lib.py

```diff
@@ -294,6 +294,36 @@ def reset_frame_range(fps: bool = True):
         frame_range["frameStartHandle"], frame_range["frameEndHandle"])
 
 
+def reset_unit_scale():
+    """Apply the unit scale setting to 3dsMax
+    """
+    project_name = get_current_project_name()
+    settings = get_project_settings(project_name).get("max")
+    unit_scale_setting = settings.get("unit_scale_settings")
```

Comment:
This could be:
```py
scene_scale = settings.get("unit_scale_settings", {}).get("scene_unit_scale")
if scene_scale:
	rt.units.SystemType = rt.Name(scene_scale)
```